resources/RELEASE_2022_05_01.md

```diff
@@ -0,0 +1,11 @@
+May the Fou-... wait no, that's in 3 days time.
+_ _
+As per [Skyra V7 plans](https://skyra.notion.site/Skyra-v7-922ba06004654142a7b63347a92513a8). We have:
+_ _
+⫸ Removed all commands from the weeb category (anime and manga will be ported later). These have been migrated to [Nekokai](https://discord.com/oauth2/authorize?client_id=939613684592934992&permissions=16384&scope=bot%20applications.commands).
+　⪢ The removed commands are: `wbang`, `wbanghead`, `wbite`, `wblush`, `wcry`, `wcuddle`, `wdance`, `wgreet`, `whug`, `wkiss`, `wlewd`, `wlick`, `wneko`, `wnom`, `wpat`, `wpout`, `wpunch`, `wsalute`, `wslap`, `wsleepy`, `wsmile`, `wsmug`, `wstare`, `wthumbsup`, and `wtickle`.
+_ _
+⫸ Released [Nekokai](https://discord.com/oauth2/authorize?client_id=939613684592934992&permissions=16384&scope=bot%20applications.commands). This bot exclusively uses slash commands, the prefix is `/`.
+　⪢ The commands are all those mentioned above, plus `awoo`, `baka`, `hand-hold`, `high-five`, `owo`, `poke`, `shrug`, `thinking`, and `wasted`.
```

Comment:
```suggestion
　⪢ The commands are `awoo`, `baka`, `bang`, `banghead`, `bite`, `blush`, `cry`, `cuddle`, `dance`, `greet`, `hand-hold`, `high-five`, `hug`, `kiss`, `led`, `lick`, `neko`, `nom`, `owo`, `pat`, `poke`, `pout`, `punch`, `salute`, `shrug`, `slap`, `sleepy`, `smile`, `smug`, `stare`, `thinking`, `thumbsup`, `tickle`, and `wasted`.
```

I just realised that otherwise people might think the `w` prefix is retained so better to list them like this